Source/UnitTests/FileUtil.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <gtest/gtest.h>
+
+#include "Common/FileUtil.h"
+
+class FileUtilTest : public testing::Test
+{
+protected:
+  FileUtilTest()
+      : m_parent_directory{File::CreateTempDir()}, file_path(m_parent_directory + "/file.txt"),
+        directory_path(m_parent_directory + "/dir"),
```

Comment:
This seems dangerous when m_parent_directory can be the empty string (if a temporary directory fails to be created for whatever reason).

Reply:
Fixed, thank you for pointing that out.  

There are a few other places in the codebase with a similar problem, but that's an issue for another PR.